Fix bfin compilation

cc1: error: unrecognized command line option "-Wunused-but-set-variable"

using this defconfig

BR2_bfin=y
BR2_PACKAGE_LIBSPANDSP=y

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

diff -uNr spandsp-0.0.6.org/configure.ac spandsp-0.0.6/configure.ac
--- spandsp-0.0.6.org/configure.ac	2014-08-09 10:19:27.000000000 +0200
+++ spandsp-0.0.6/configure.ac	2015-04-25 17:46:27.717349873 +0200
@@ -337,7 +337,7 @@
 
 case "${ax_cv_c_compiler_vendor}" in
 gnu)
-    COMP_VENDOR_CFLAGS="-std=gnu99 -ffast-math -Wall -Wunused-variable -Wunused-but-set-variable -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes $COMP_VENDOR_CFLAGS"
+    COMP_VENDOR_CFLAGS="-std=gnu99 -ffast-math -Wall -Wunused-variable -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes $COMP_VENDOR_CFLAGS"
     if test "$enable_neon" = "yes" ; then
         COMP_VENDOR_CFLAGS="-mfpu=neon -mfloat-abi=hard $COMP_VENDOR_CFLAGS"
     fi
